build.gradle

```diff
@@ -101,6 +101,7 @@ subprojects {
         testImplementation ("org.junit.jupiter:junit-jupiter-params")
         testRuntimeOnly ("org.junit.jupiter:junit-jupiter-engine")
         testRuntimeOnly ("org.junit.platform:junit-platform-launcher")
+        testImplementation ("org.assertj:assertj-core:3.26.3")
```

Comment:
Approved added dependency:
1) It is a dependency for testcases only and does increase runtime dependencies
2) It adds clarity to test cases